Xamarin.Forms.Core/PlatformConfiguration/iOSSpecific/ShellSearchHandler.cs

```diff
@@ -0,0 +1,27 @@
+﻿using FormsElement = Xamarin.Forms.Shell;
```

Comment:
`IPlatformElementConfiguration` can only be applied to the type `Element`. `SearchHandler` isn't Element. Because of this I had to add to the `Shell` page. That is, all `SearchHandler` elements on the shell page will depend on this platform specific property.